gallery/src/components/NxCombobox/NxComboboxPage.tsx

```diff
@@ -176,6 +180,30 @@ const NxComboboxPage = () =>
                 parameter and should return a string or <NxCode>ReactNode</NxCode> specifying the tooltip content.
               </NxTable.Cell>
             </NxTable.Row>
+            <NxTable.Row>
+              <NxTable.Cell><NxCode>filterInput</NxCode></NxTable.Cell>
+              <NxTable.Cell><NxCode>true</NxCode> | <NxCode>false</NxCode> | <NxCode>"search"</NxCode></NxTable.Cell>
+              <NxTable.Cell>No</NxTable.Cell>
+              <NxTable.Cell>false</NxTable.Cell>
+              <NxTable.Cell>
+                <NxP>
+                  Normally, <NxCode>NxCombobox</NxCode> uses a plain <NxCode>NxTextInput</NxCode> for its text input.
+                  However, <NxCode>NxCombobox</NxCode>'s functionality can also be useful in places where the styling of
+                  a filter input is desired – cases where <NxCode>NxFilterInput</NxCode>'s{' '}
+                  <NxCode>datalist</NxCode> prop would have formerly been used. To enable use
+                  of <NxCode>NxCombobox</NxCode> in such cases, it has a <NxCode>filterInput</NxCode> prop which causes
+                  it to use an <NxCode>NxFilterInput</NxCode> internally rather than
+                  in <NxCode>NxTextInput</NxCode>. When this prop is true, an <NxCode>NxFilterInput</NxCode> with the
+                  default "filter" icon is rendered. When this prop is instead set to "search",{' '}
+                  an <NxCode>NxFilterInput</NxCode> with a "search" icon is rendered instead.
+                </NxP>
+                <NxWarningAlert>
+                  <NxCode>NxFilterInput</NxCode> does not support validation. Therefore,
```

Comment:
Out of curiosity, will it support validation in the future?

Reply:
> Out of curiosity, will it support validation in the future?

There aren't plans for it to.  The places where a filter input would be used and the places where validation would be used would seem to be completely separate.